lib/Sema/CSSimplify.cpp

```diff
@@ -9513,14 +9517,47 @@ performMemberLookup(ConstraintKind constraintKind, DeclNameRef memberName,
   if (isSingleUnlabeledPackExpansionTuple(instanceTy)) {
     auto elementTy = instanceTy->castTo<TupleType>()->getElementType(0);
     if (elementTy->is<TypeVariableType>()) {
-      MemberLookupResult result;
       result.OverallResult = MemberLookupResult::Unsolved;
       return result;
     }
   }
 
+  // Delay solving member constraint for unapplied methods
+  // where the base type has a conditional Sendable conformance
+  if (ctx.LangOpts.hasFeature(Feature::InferSendableMethods)) {
+    if (functionRefKind == FunctionRefKind::Unapplied) {
```

Comment:
Shouldn't this use `isPartiallyApplied` check as well?

Reply:
I think this is the most important to get right before we merge this PR.

Reply:
fixed!